pom.xml

```diff
@@ -30,14 +30,14 @@
     <!-- properties for script build step that creates the config files for the artifacts -->
     <widgets.dir>widgets</widgets.dir>
     <docs.dir>docs</docs.dir>
-    <app.parents>system:cdap-data-pipeline[6.1.1,7.0.0-SNAPSHOT)</app.parents>
+    <app.parents>system:cdap-data-pipeline[6.8.0,7.0.0-SNAPSHOT)</app.parents>
     <!-- this is here because project.basedir evaluates to null in the script build step -->
     <main.basedir>${project.basedir}</main.basedir>
     <jee.version>7</jee.version>
     <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
-    <cdap.version>6.1.1</cdap.version>
-    <cdap.plugin.version>2.3.4</cdap.plugin.version>
-    <hadoop.version>2.9.2</hadoop.version>
+    <cdap.version>6.8.0</cdap.version>
```

Comment:
Are we able to bump the hadoop version without bumping the CDAP version?

Reply:
Reverted other dependency bump and only bumped hadoop dependency version.